getting-started/data-node-authentication.md

```diff
@@ -0,0 +1,70 @@
+# Data Node Authentication
+
+In a multi-node environment, it is necessary to ensure that the access
+node have the necessary roles and permissions on all the data nodes
+where distributed hypertables are placed.
+
+This is critical since when TimescaleDB is executing a query or
+command it will use the same role on the data nodes for the
+fragments. This means that the same role need to exist on the data
```

Comment:
```suggestion
This means that the same role needs to exist on the data
```
not sure I understand "fragments". We've never used this terminology before and it is confusing to me.

I think you can just delete "for the fragments".